js/source/view-counter.js

```diff
@@ -1,30 +1,27 @@
-(function () {
-    var request = new XMLHttpRequest();
-    var span = document.getElementById('view-counter');
-    if (!span) {
-        return; // abort if no view-counter element is found
-    }
-
-    var onLoad = function () {
-        if (request.status !== 200) {
-            return; // don't do anything on request error
-        }
+(async function () {
+  const span = document.getElementById("view-counter");
+  if (!span) {
+    return; // abort if no view-counter element is found
+  }
 
-        var body = JSON.parse(request.responseText);
+  const response = await fetch(
```

Comment:
**thought**: probably not worth a polyfill

![image](https://user-images.githubusercontent.com/325813/114105470-09aa5a00-988a-11eb-9927-cd495ab72764.png)

Reply:
I agree. If fetch isn't available, the function throws an error and the counter never shows up. The user would never know. Still, that is crazy how much IE there still is out there.